### ISSUE TITLE
Add Favicon

### ISSUE THREAD

**Issue by thansidwell** [CodeForPhilly/vacant-lots-proj#186]
As a user I want to be able to see the favicon with the Clean & Green logo on my browser tab. Right now it has something else there. https://share.cleanshot.com/ysvvtJ8p 

1. Add the new favicon based on the icon version of the logo. https://www.figma.com/file/NAFkgq34abW6uJ0R7PW24T/Prototype---Clean-%26-Green-Philly?type=design&node-id=1326%3A13797&mode=design&t=h1fTrSpJsuOpTu9c-1
2. I added a square frame around it and exported as a SVG here. [Favicon - Clean & Green Philly.svg.zip](https://github.com/CodeForPhilly/vacant-lots-proj/files/13958260/Favicon.-.Clean.Green.Philly.svg.zip)
3. I also ran it through https://realfavicongenerator.net/ to generate all the various files. I can't remember is all this is overkill or not. Every browser and OS has different favicon and the last time I did it just used this tool. Here's the exported package. [favicon_package_v0.zip](https://github.com/CodeForPhilly/vacant-lots-proj/files/13958254/favicon_package_v0.zip)
4. It included these instructions 

```
1. Download your package:  (compression rate: 75%)
2. Extract this package in the root of your web site. If your site is http://www.example.com, you should be able to access a file named http://www.example.com/favicon.ico.
3. Insert the following code in the <head> section of your pages:
<link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
<link rel="manifest" href="/site.webmanifest">
<link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
<meta name="msapplication-TileColor" content="#ffffff">
<meta name="theme-color" content="#ffffff">
```

**Comment by 19ewalker** [comment]
hey, just wondering, would adding just the normal favicon.ico be sufficient to close this ticket, or does it need to be the whole package to meet the criteria? I have the new favicon.ico working when I add that to the metadata, but am having issues with the package's icons showing up when using the HTML link tags as above in the instructions

**Comment by thansidwell** [comment]
@19ewalker Yeah, good call. I'm happy if just the favicon works. The other icons are less commonly used and I thought if we could easily do it great, but if we run into issues lets just skip the others. It's not worth our time right now.

**Comment by 19ewalker** [comment]
alright, sounds good! if you can assign the ticket to me, I'll make the PR

**Comment by thansidwell** [comment]
@19ewalker You can't assign it to yourself??

**Comment by nlebovits** [comment]
lol @19ewalker I assigned it to you

**Comment by nlebovits** [comment]
closed by #242
